README.md

```diff
@@ -3,44 +3,45 @@ python-systemd
 
 Python module for native access to the systemd facilities. Functionality
 is separated into a number of modules:
-- systemd.journal supports sending of structured messages to the journal
+- `systemd.journal` supports sending of structured messages to the journal
   and reading journal files,
-- systemd.daemon wraps parts of libsystemd useful for writing daemons
+- `systemd.daemon` wraps parts of `libsystemd` useful for writing daemons
   and socket activation,
-- systemd.id128 provides functions for querying machine and boot identifiers
+- `systemd.id128` provides functions for querying machine and boot identifiers
   and a lists of message identifiers provided by systemd,
-- systemd.login wraps parts of libsystemd used to query logged in users
+- `systemd.login` wraps parts of `libsystemd` used to query logged in users
   and available seats and machines.
 
 Installation
 ============
 
 This module should be packaged for almost all Linux distributions. Use
 
-On Fedora/RHEL/CentOS
+On Fedora:
 
-    dnf install python-systemd python3-systemd
+    dnf install python3-systemd
 
-On Debian/Ubuntu/Mint
+On Debian/Ubuntu/Mint:
 
-    apt-get install python-systemd python3-systemd
+    apt-get update
+    apt-get install python3-systemd
```

Comment:
```suggestion
    apt update
    apt install python3-systemd
```
would be the more modern way of putting this, although I'd omit the `apt update`. It will complain (nicely) if the sources are out of date.

Reply:
Isn't the problem that `apt` isn't available everywhere yet? (I prefer it too, but I thought that `apt-get` is more widely supported…)

Reply:
> although I'd omit the apt update. It will complain (nicely) if the sources are out of date.

No, it doesn't:
```console
$ podman run -it --rm debian
root@adad5c659058:/# apt install python3-systemd
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package python3-systemd
```
(And after `apt update`, it works.)

Reply:
`apt` is available in the stretch, which is the oldest [listed release](https://packages.debian.org/bullseye/apt) for it and stretch has already ended LTS support, so I don't think it's super relevant. There might of course still be super LTS stuff flying around, but I wouldn't think they are a relevant target audience.

Reply:
Updated.